cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/Broker.java

```diff
@@ -631,21 +641,20 @@ public Disk disk(String logdir) {
   }
 
   /**
-   * Get per-logdir disk utilization information of the broker.
+   * Get per-logdir disk utilization information for all alive disks of the broker.
```

Comment:
Can we also 
1. update the `return` tag to say that this function is relevant only when the clusterModel has been created with a request to populate replica placement info, and otherwise returns an empty map -- i.e. not `null` as indicated currently.
2. rename the function to `aliveDiskUtils()`?